packages/server/modules/workspaces/domain/logic.ts

```diff
@@ -59,3 +59,8 @@ export const parseDefaultProjectRole = (
 
   return role
 }
+
+export const isWorkspaceRole = (role: string): role is WorkspaceRoles => {
```

Comment:
i know this is trivial implementation, but still, i'd like to add unit tests for this, that tests the behavior